Added output file for option 3

KevinFlynn10 opened the pull request:
When option #3 is run, the program will output everything that it couts to the file called output.txt. If it isn't run, the file will just be blank.

austinholler commented:
Hello Kevin,

There's a problem with the code and I'm not quite sure why. It seems like it should work, but when choosing option 3 it's unable to go through the entire file without seg faluting. Commenting out the writing code it's able to run fully, but while writing out the info it doesn't like something.

If you can figure it out feel free to request a pull again. Thank for the time and help!

KevinFlynn10 commented:
Hi Austin,

Was there an exact error when trying to run it besides a seg fault? Also did you use all three update files main.cpp, SNP_fun.cpp, and SNP_Fun.h? I do not get any seg faults when running and all the info for 3 is output onto the file. I just ran the program from codeblocks, entered 3 and put in test.txt, then 6 and it worked. I'm not sure why this is happening as there really wasn't much I added. Just changed retrieveData to also output to file when couting and then opened and closed it in main. Not sure why that would seg fault. If I found a way to reproduce what happened to you id try to help, but I'm not exactly sure what's happening. Lemme know if you figure anything out!

> On May 2, 2015, at 7:56 PM, austinholler notifications@github.com wrote:
> 
> Hello Kevin,
> 
> There's a problem with the code and I'm not quite sure why. It seems like it should work, but when choosing option 3 it's unable to go through the entire file without seg faluting. Commenting out the writing code it's able to run fully, but while writing out the info it doesn't like something.
> 
> If you can figure it out feel free to request a pull again. Thank for the time and help!
> 
> —
> Reply to this email directly or view it on GitHub.
